### DISCUSSION TITLE
Seeking Open-Source Game Engines with BehaviorTree.CPP Integration

### DISCUSSION THREAD

**Discussion by fereydoon031** [BehaviorTree/BehaviorTree.CPP#645] [Q&A]
Hey everyone,
I'm on the lookout for open-source game engines that have incorporated BehaviorTree.CPP . I haven't come across any such projects on GitHub so far. Can anyone share if there are repositories or projects out there that make use of BehaviorTree.CPP for behavior implementation? Your insights and pointers would be really helpful!

**Comment by Alex0vSky** [comment]
I have a suspicion that state machines are a purely academic discipline. Just to train your mind.
But I don't want to believe it.
We need to remain determined.
